src/components/project-tabs/QRTab.jsx

```diff
@@ -275,7 +275,7 @@ const QRTab = (props) => {
           >
             <InputGroup className="input-group-prepend">
               <FormControl
-                ref={inputEl}
+                // ref={inputEl} Can't have inputEl in both places
```

Comment:
The second occurrence of `InputEl`.  I disabled it so the desktop version submit would work properly.

Reply:
I can see the input field codes are actually duplicating, it is possible to remove any duplicate code and control the content to display on different devices using an internal state?